pygfx/renderers/wgpu/engine/blender.py

```diff
@@ -535,6 +536,9 @@ class BaseFragmentBlender:
     Each renderer has one blender object.
     """
 
+    # Each blender should define a unique name so that it may be
+    # correctly registered in the renderer.
+    name = None
```

Comment:
Sorry for the "major" refactor here but lmk what you think. I think it makes it cleaner to define the name of the blender as part of the blender class not "the name of the blender as the renderer believes it to be".

It does create the possibility of clashes downstream, but those issues tend to resolve themselves by the consuming libraries.

Reply:
I think this makes sense. As a side-effect, the decorators can also omit the `()` so the implementation of `_register_blend_mode` can be simpler too :)

Reply:
is this the syntax you had in mind?